TESTS/mbed_drivers/lp_timeout/main.cpp

```diff
@@ -74,9 +74,10 @@ Case cases[] = {
     Case("1 s delay during deepsleep (attach_us)", test_deepsleep<AttachUSTester<LowPowerTimeout>, 1000000, LONG_DELTA_US>,
          greentea_failure_handler),
 #endif
-
+#if !defined(__ARM_FM)  //FastModels not support time drifting test
```

Comment:
Is it OK to remove some tests because they are not passing....?

Reply:
Not sure how is it not supported?

@ARMmbed/mbed-os-hal to review

Reply:
It is same story as #10030

For partner's targets/platforms, which will be used real life production. we are not allow them to skip tests, if they can't make them pass. The tests are mean to check that all platform comply with mbed HAL API.

FastModel it is a software simulator for Arm platform/cores. mean to be used as a tool for testing or early stage development. It is will never be used for any real IoT products. 
They can simulator how ARM core works. One of issue for software simulator is they running on your host PC, they can't grantee the timing accuracy as microelectronics.

So the exception is just for the software models